app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java

```diff
@@ -107,6 +120,31 @@ protected void onBindViewHolder(@NonNull final BookViewHolder holder, int positi
                             book.getAuthor());
                     holder.statusTextView.setText(
                             book.getStatus().toString());
+
```

Comment:
Weird thing I notice here is that the book list in the owner page fragment shows the wrong borrower. The book info page shows the correct borrower and the borrower is reflected in Firebase but not in the owner page fragment.

Screenshots:
[Owner Page Fragment](https://drive.google.com/open?id=1YlYhiDdYvGG0S7G3yy4HrbyxCfG-Z2ON)
[Book Info for Cal Newport Deep Work](https://drive.google.com/open?id=1r9LbxBwYk6NR2VAqw2VtTiKGPipj07tP)
[Firebase Data](https://drive.google.com/open?id=15lX9nnsoAdbkvwwrsiGO8kMoAIsCOCEY)

Reply:
Fixed by changing to standard firebase ref calls.

Reply:
Yeah looks like a problem with the fact that ViewModels are like a "mini-singleton" in that there can only be one instance per activity. So I don't think we can create multiple instances of the same ViewModel class in an activity.

Reply:
That will have to be investigated for later